tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_resource_tests.go

```diff
@@ -4,18 +4,12 @@ import (
 	"encoding/json"
 	"strings"
 
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/components/dataapigeneratorjson/models"
```

Comment:
for consistency (given there's an existing `models` package within `importer-rest-api-specs`) can we alias this import:

```suggestion
	dataApiModels "github.com/hashicorp/pandora/tools/importer-rest-api-specs/components/dataapigeneratorjson/models"
```